src/components/DatePicker/DatePicker.js

```diff
@@ -212,14 +213,14 @@ export default class DatePicker extends Component {
     locale: 'en',
   };
 
-  UNSAFE_componentWillUpdate(nextProps) {
-    if (nextProps.value !== this.props.value) {
+  componentDidUpdate(prevProps) {
+    if (!isEqual(prevProps.value, this.props.value)) {
       if (this.cal) {
-        this.cal.setDate(nextProps.value);
+        this.cal.setDate(this.props.value, true);
```

Comment:
Wondering why the 2nd argument (`triggerChange`) is added here - The addition caused `onChange` event firing even for non-user-initiated change.

Reply:
@asudoh 
> Seems like this is a known bug in the flatpickr library (flatpickr/flatpickr#1345), but during investigation I've discovered that in our case this can be fixed by forcing the calendar to update using optional second argument in setDate. 

I couldn't fix it without adding it, I think that its a bug in rangeplugin in flatpickr and the second calendar doesnt update when using a setDate without the second argument.